src/util/BaseLoginController.js

```diff
@@ -92,8 +92,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+    // Event triggered at after a page is rendered along with the classname to identify the page
+    postRender: function() {
```

Comment:
I added the postRender to handle cases where there are animations on the page .
I had taken care of controllers calling their parent class in another commit. I'll rectify that now